packages/smooth_app/lib/helpers/network_config.dart

```diff
@@ -13,48 +14,31 @@ Future<void> setupAppNetworkConfig() async {
 }
 
 Future<void> _initUserAgent() async {
-  final PackageInfo packageInfo = await PackageInfo.fromPlatform();
+  try {
+    final PackageInfo packageInfo = await PackageInfo.fromPlatform();
 
-  final String name = 'Smoothie - ${packageInfo.appName}';
-  final String version = '${packageInfo.version}+${packageInfo.buildNumber}';
-  final String system =
-      '${Platform.operatingSystem}+${Platform.operatingSystemVersion}';
-  final String comment = _getAppInfoComment(
-    name: name,
-    version: version,
-    system: system,
-  );
-  OpenFoodAPIConfiguration.userAgent = UserAgent(
+    final String name = 'Smoothie - ${packageInfo.appName}';
+    final String version = '${packageInfo.version}+${packageInfo.buildNumber}';
+    final String system =
+        '${Platform.operatingSystem}+${Platform.operatingSystemVersion}';
+
+    // Setting no comment here as this will be overridden at runtime
```

Comment:
> Setting no comment here as this will be overridden at runtime

By whom?

Reply:
By setUserAgentComment in product_query.dart

Currently, it's either "Scan" or "Search"

Also the _getAppInfoComment just provided the same info as the normal values so there was no added value of it